HeinOnline.js

```diff
@@ -0,0 +1,240 @@
+{
+	"translatorID": "3dcbb947-f7e3-4bbd-a4e5-717f3701d624",
+	"label": "HeinOnline",
+	"creator": "Frank Bennett",
+	"target": "https?://heinonline.org/HOL/(?:LuceneSearch|Page|IFLPMetaData)\\?",
+	"minVersion": "1.0",
```

Comment:
since it's using `ZU.xpath`, this should be 3.0. I don't think we do anything with that info, but might as well do it right.